src/views/user/double-authentication-choice.ejs

```diff
@@ -47,6 +47,10 @@
           </label>
         </div>
       </div>
+      <div class="fr-checkbox-group fr-pl-1w fr-pb-2w">
+        <input type="checkbox" name="2fa_force" id="mfa-proconnect-config" >
```

Comment:
suggestion: name, id and label could share the same value "force_2fa" for better readability